nimbus-test/src/test/java/com/antheminc/oss/nimbus/test/scenarios/s11/FunctionExecutorTests.java

```diff
@@ -48,12 +48,7 @@
 	@Test
 	@SuppressWarnings("unchecked")
 	public void t01_initEntity() {
-		MockHttpServletRequest request = MockHttpRequestBuilder.withUri(PLATFORM_ROOT+"/sample_functest")
-					.addAction(Action._new)
-					.addParam("fn", "_initEntity")
-					.addParam("target", "parameter1")
-					.addParam("json", "\"Test\"")
-					.getMock();
+		MockHttpServletRequest request = MockHttpRequestBuilder.withUri(PLATFORM_ROOT+"/sample_functest/_new?fn=_initEntity&target=/parameter1&json=\"Test&1234\"").getMock();
```

Comment:
Did we have assertion to ensure `"Test&1234"` is set?

Reply:
Good point, missed that. Let me add that